client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/style.scss

```diff
@@ -10,7 +10,7 @@
 		display: none;
 	}
 	.step-container {
-		background: var( --studio-gray-0 );
+		background: #fdfdfd;
```

Comment:
Was there a reason we switched from a css variable to a hard coded hex code?

Reply:
I couldn't find a css variable for this hex code, so I didn't convert it to a variable.